src/sentry/features/__init__.py

```diff
@@ -182,6 +182,7 @@
 default_manager.add("organizations:discover-basic", OrganizationFeature)
 default_manager.add("organizations:discover-query", OrganizationFeature)
 default_manager.add("organizations:dynamic-sampling", OrganizationFeature)
+default_manager.add("organizations:ds-prioritise-by-project-bias", OrganizationFeature)
```

Comment:
As per
https://github.com/getsentry/sentry/blob/136f4a4cd2e3f54d138fb5eb6c5d7d7cc49ac856/src/sentry/features/__init__.py#L169-L173
This section is reserved for flags controlled by SubscriptionPlanFeatureHandler.
Please put it alphabetically in the section above this one.

Reply:
@matejminar fixed in https://github.com/getsentry/sentry/pull/43498/commits/03207ec398d279d7274eccd5cf9419d40f3ae3f2